Aptoide/aptoide-client-v8
# [ASV-1757] Helping wallet install metrics

**What does this PR do?**

   This PR aims at implementing the metrics needed to the helping AppCoins wallet installation feature.
Also fixed the context used on the download complete event that is sent facebook. Previously we were using the previousContext instead of the currentContext meaning that we are going to send the previous view instead of the current view where the download is starting. 

Also fixed the Install and Download events sent to the BI by adding the appc flags, which were not added in ASV-1591.

**Database changed?**

No

**Where should the reviewer start?**

- [ ] WalletInstallAnalytics.java

**How should this be manually tested?**

  Perform wallet download and installation and see the events being sent in both Charles (BI) and analytics manager logs.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1757](https://aptoide.atlassian.net/browse/ASV-1757)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
